### PR TITLE
Tracking and notifying participants...

### PR DESCRIPTION
**Pull request by elight**
... while trying not to notify the user who caused the event.

### CODE REVIEW

**Comment by elight** [comment]
Is this thing on?

**Comment by awwaiid** [comment]
maybe

**Comment by elight** [comment]
Maybe I won't get notified now?

**Comment by elight** [comment]
Or it will all break

**Comment by elight** [comment]
Nope that's not right

**Comment by elight** [comment]
Ok, let's see if it only notifies Brock this time...

**Comment by awwaiid** [comment]
Brock says hi

**Comment by elight** [comment]
@awwaiid Yes, I noticed.

File: src/github/message_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          @msg.action
+        end
+    end
+
+    def interested_slack_users
+      topic  = "gh-#{@msg.pr}"
+      user   = @msg.commenter    || @msg.sender
+      author = @brain.slack_name_for(user)
+
+      @brain.remember_participant(author, in_discussion: topic)
```
Comment (elight) [inline src/github/message_presenter.rb]: I will go mad if I don't get this line and much of the following lines into a different abstraction.

**Pull request merged**